mw-webapp/src/component/avatar/Avatar.tsx

```diff
@@ -69,26 +70,51 @@ const getInitials = (name: string): string => {
  */
 export const Avatar = (props: AvatarProps) => {
   const initials = getInitials(props.alt);
+  const location = useLocation();
 
-  return (
-    <AvatarRadix.Root
-      className={clsx(styles.AvatarRoot,
-        styles[props.size ?? AvatarSize.SMALL],
-        props.className)}
-      data-cy={props.dataCy}
-    >
-      <AvatarRadix.Image
-        className={styles.AvatarImage}
-        src={props.src ?? ""}
-        alt={initials}
-      />
-      <AvatarRadix.Fallback
-        className={styles.AvatarFallback}
-        delayMs={600}
+  if (location.pathname === "/") {
```

Comment:
super dangerous, avatar as component should not change it behavior in depends on the location

Reply:
Agree, discused about it in telegram